project/src/components/Nav.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react'
+import { connect } from 'react-redux'
+import { Router, Route, IndexRoute, browserHistory } from 'react-router'
+import {Link, withRouter} from 'react-router-dom'
+import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import '../css/style.css';
+import $ from 'jquery'
+
+const {onClickNews, onClickMain, onClickFriends, onClickSettings, onClickSearch} = require('../constants');
+const SendRequest = require ('../components/Requests.js');
+
+const UserBlockRouteComponent = withRouter(
+    class UserBlock extends React.Component{
+        onClickMain (e, user){
+            e.preventDefault();
+            SendRequest.sendForMain(onClickMain, 'GET', user, this.props);
+        }
+
+        onClickFriends (e, user){
+            e.preventDefault();
+            SendRequest.sendForFriends(onClickFriends, 'GET', user, this.props);
+        }
+
+        onClickSearch(e, user){
+            e.preventDefault();
+            SendRequest.sendForSearch(onClickSearch, 'GET', user, this.props);
```

Comment:
Didn't get, why do you need to send to link to the current function to helper-function?